tests/containers/podman_remote.pm

```diff
@@ -9,8 +9,33 @@
 
 use Mojo::Base 'containers::basetest';
 use testapi;
-use utils qw(script_retry systemctl);
+use utils qw(script_retry systemctl zypper_call);
 use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use transactional qw(trup_call check_reboot_changes);
+use version_utils qw(is_transactional is_sle is_sle_micro);
+use containers::utils qw(get_podman_version);
+
+my $podman_version;
+
+sub test_package {
+    my $ret;
+
+    if (is_transactional) {
```

Comment:
```suggestion
    # Allow installation failure so we can check for bsc#1126596 later on.
    if (is_transactional) {
```
It took me some time to realize why we allow the installation to fail. Let's put a comment here.